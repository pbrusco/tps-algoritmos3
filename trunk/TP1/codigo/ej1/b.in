206204	932357
-834545	613348
341485	896979
417394	489056
986393	764935
986606	29454
346452	204122
-414981	561990
883270	68166
207899	605533
-291768	328089
-4877	162634
-94808	20471
-710662	437865
785202	940611
-605127	443569
411793	790022
-370126	721355
-736120	120977
485707	328876
686500	620644
852353	141874
-115202	236682
791381	463696
472968	248898
435841	370377
-911815	782170
624614	668649
899403	404769
242357	406828
-223547	93328
-532825	945682
4737	577236
-13471	368618
976421	357938
-879982	793779
710759	221946
-364261	846560
-222765	262316
202245	21025
-26952	244573
-472303	777398
575879	298487
-916136	311958
236440	804732
-321994	684714
-777924	911825
532375	276086
471827	498851
-299695	701096
-50742	244400
-411469	233056
-717877	808935
-117873	241424
-402837	994216
562091	316210
-515122	610486
-690099	659214
227987	131041
28190	430736
-365769	997830
947921	409300
-307470	643529
481857	277754
-968202	196944
775630	759035
267771	274157
-671736	964256
351786	708595
530686	253137
-927033	618907
-84132	83180
474285	907002
189725	388859
146588	873414
-130371	649044
-430363	916816
-340840	588552
-426883	940339
-643965	987377
808135	430763
970134	31247
-57784	505532
305314	211610
830598	874550
-552825	4921
-34086	435284
305856	776125
-541881	203008
-2852	363325
716960	171460
82678	657946
15223	232082
-371121	537397
632230	884347
250399	437173
33964	987611
250696	293467
55980	351700
-317953	354553
-83655	587865
-280119	670544
507829	202119
757423	89593
218625	721823
70729	488574
411402	38891
-703710	289587
-772142	345567
-76285	663521
-743752	747176
-582679	543648
595904	51477
-841210	325252
-140264	543878
-323664	614607
460589	542361
585124	762423
-519477	534565
-805530	127202
-446135	870955
-707309	453634
639025	49538
289212	890748
28991	547364
-523296	871029
-506175	847970
527943	433094
839298	468923
-12572	274454
365717	720589
982390	427898
142710	583276
-870859	872488
-924259	417832
521536	941128
766123	447303
-553071	491599
102442	330897
-219081	859821
946569	741891
61364	240633
629792	383344
-280461	770555
821878	694814
-910659	732702
-903171	498825
-227301	568249
-17227	670691
276076	406124
809378	352693
579621	930409
-777761	76553
75528	357015
762134	178893
-866572	605904
912137	509076
437708	252729
938526	269956
286592	546033
-10830	871763
476959	451384
183720	745497
-790176	337377
-683538	99512
-755361	966084
-76237	394573
895937	348633
283886	287159
-858876	90103
-839990	100934
206500	94245
-987066	794317
844185	584493
574670	784383
-484048	539744
999329	615981
540037	28270
-817121	312156
-880512	687385
6288	43727
-3826	250227
-969099	237293
-679869	597831
-728016	172501
-392025	172901
538470	688582
79486	228619
-485487	562092
-178647	442605
-478261	965245
-878808	485423
-373599	454522
-662706	650744
-569009	378760
743342	287137
973770	825607
336530	421446
727559	906933
947096	601932
370369	596545
837368	475353
554593	848952
-25463	28010
610514	597019
-368066	856713
814871	830483
950257	683365
-830613	410925
756709	874373
-158113	244743
524761	82111
893183	153057
716127	178520
-706165	305386
74963	673453
247249	4676
-245141	471285
-664479	301898
-23453	574960
984358	249425
541808	290539
574611	183722
972717	416201
-915184	122367
-454670	713682
-850869	477283
593902	722424
536651	903255
-890325	926708
118371	427419
-623863	969227
-211465	543838
952124	32907
-387653	464444
-128568	919114
59287	286335
994681	396589
-917711	933241
658991	823566
-491581	458289
-89948	82152
462501	809969
-535128	278445
540307	666099
-667852	311019
509078	370307
-343938	881340
-417573	315404
-394463	490747
330765	498680
178132	588629
-994726	567482
387862	102610
813904	159269
704389	827121
-30339	852551
-642624	712841
167850	130415
-626315	41230
-424698	371995
-681368	66480
-536542	61206
718036	965420
177974	779324
304011	65
-257418	546757
-581131	705733
302535	873583
339343	16251
-866896	440949
893129	638669
-507990	691563
690333	409599
7989	103926
-968561	407937
272606	181707
945554	279190
331342	581725
-847544	921068
-437679	787965
-399809	681094
-428418	189085
34585	395770
606171	920111
-989784	405024
-857110	677631
-787544	139537
940063	470879
-966164	318423
524867	756102
524518	672264
679189	100682
9071	651619
-781256	774142
-141013	763927
941411	621037
407162	924934
668455	381349
27824	347513
791908	388732
376661	913250
-126155	592439
97437	117863
-980840	899119
784353	556484
992889	497895
-879056	421410
421110	89865
434043	117689
281749	425949
-401909	802610
677980	445117
87437	58906
632160	39746
951024	824099
550167	816989
286698	212397
-948162	149859
-507326	100254
-930354	382004
225761	783913
-467121	978246
300919	65683
-768932	214195
-652414	681571
-92380	231738
-546593	518436
-422214	982950
904052	6628
-19929	936982
561755	679095
-938895	146216
761151	963488
158664	732420
595923	901186
373633	993567
484682	56512
-324480	478726
-314136	382778
920397	402708
908662	480815
197191	419710
980317	697213
-829125	855878
408479	451801
-153051	341786
-645713	826469
689515	667301
199914	981437
-233292	418186
203652	326848
-853107	40391
165129	20708
-440638	366185
448566	774664
874031	444068
809415	606133
801309	295648
271929	11914
641679	761559
-930487	965211
-700110	334670
-922147	16151
-841396	456789
-616522	905355
-359800	295739
444864	105154
356396	422815
451580	694745
-44552	852776
-539358	783263
393715	999726
-288462	921873
720269	279621
518261	896143
-607268	255943
-656694	217160
312252	89908
779810	541489
259282	586041
-16663	641751
739255	551818
-312869	840281
-902633	560550
-465537	595163
-933796	718783
-697793	375477
599647	204082
924316	983892
943728	759526
332268	292542
-20422	548149
708301	861018
834404	280003
756479	745541
574696	679337
-648855	893483
-456526	493130
837752	933798
793170	877527
-334628	209795
34189	230217
-285568	454870
893578	805626
-62138	562105
-825861	136801
-308366	785656
-682699	758534
-261114	112638
339772	905808
694307	756789
341384	307330
52961	109250
-625555	2829
478152	64967
592641	407180
902819	715546
622830	914598
-835971	692064
-499262	31836
40078	242495
307041	100231
320571	153192
-788971	778748
-62995	773252
415476	882246
-904756	301417
-696044	924247
245866	276571
-908325	775833
992943	332263
350702	639304
-499438	959876
273775	748847
-50941	328194
-409425	743670
175088	648427
172649	344471
84807	590337
689028	15014
325084	7958
-236339	875012
247824	374450
-412500	164577
-872276	731870
-621655	657647
736927	832736
669967	521737
-43605	606545
-210166	295573
-152558	137010
427213	373349
-373876	621173
280413	33673
452149	422301
952078	43956
712028	297235
-120405	483554
389604	43512
765491	253678
156392	406236
-71602	349802
-861823	723678
-542845	4091
388444	456240
998616	924670
750082	636699
366988	273456
505630	663060
-250816	944903
807926	101296
-985694	172898
53161	34722
-741801	577567
980465	482363
-157781	480979
-862701	747414
-897560	630754
899620	136384
-461543	903552
-760791	711478
982626	213524
70615	783038
-365928	524839
-698036	21656
2736	179437
232105	558491
-614524	456051
270398	355671
-622035	333566
402222	94357
800057	593335
-370789	663950
492594	546231
708317	760619
-406403	763355
585468	511812
28297	126336
101880	913086
311845	535121
-458066	937344
-315764	253753
-903337	140894
-273948	633488
-685748	858157
-119204	264561
248448	366381
-936182	394678
-79966	496558
612124	324756
-961495	299174
921574	614938
991339	34627
-208702	824927
4999	27028
45563	146232
365833	911032
681562	847214
156331	443532
-544186	55656
983513	17151
-360183	455077
506957	446416
-794624	171471
-232130	176470
527792	222033
-88015	587866
530969	785781
-884953	458464
735495	2650
-427143	986163
-934317	862698
-56427	886007
-729516	196983
-227532	429113
752574	473258
-456552	77625
555208	608594
533782	493548
955886	745395
-433646	688890
444907	623207
-427597	195987
856702	925503
973629	669387
-267025	421962
782454	394866
402101	950074
-323879	208
704121	956095
87013	389741
972419	351000
236874	294949
903688	668003
-767485	641632
-576746	908658
859264	207464
-219406	125917
-704493	449797
199480	153918
439860	757283
885482	246054
954311	482929
735477	902969
-442146	186806
818202	763552
-132570	622816
-841804	358574
-17463	63068
-676977	262548
-808938	218760
-846990	104243
-236042	574906
-436395	310383
-304750	268882
-814736	87084
749732	219654
408543	577810
-193270	595273
944146	788603
-365291	113893
-543718	960883
-968199	713278
391960	666025
235042	970775
-264929	785511
58367	51595
507325	460138
-314857	169761
185455	113907
351305	995550
735914	539268
741117	507468
-321225	415780
-728067	650822
-920143	915751
199301	974119
698113	997796
807439	312653
512037	14461
5237	365766
-612329	618032
213007	359149
-526963	196726
-506942	924793
457696	844936
-710312	44238
-14522	258703
295222	582494
-745682	94531
-310048	99768
386315	228449
-1850	441457
-124851	968420
179984	991714
885939	449410
507644	676074
-619819	206949
-729789	502171
-668764	247854
-105847	74254
-941399	460570
-878630	462420
-207713	103623
217169	283608
667770	685899
-479658	193543
-335783	329714
-483314	575855
-696987	244620
521543	350467
445866	808219
-598637	203201
709217	410915
788215	628084
-375279	812206
-411753	291864
-157004	143999
592851	627314
-47220	324301
749443	362196
42271	808062
795668	406700
-11484	632269
-330922	936836
450320	312116
255615	240221
-770805	397225
615707	506428
-36464	553648
-218192	819443
-750387	861714
90799	657382
-150764	185218
-105291	516141
-720606	482813
631774	254780
351157	25585
214539	157645
-221586	710461
552934	928654
-246477	679041
748081	286192
689089	436957
-746012	58600
-256123	779206
-594701	927332
-657147	278490
-633169	9381
-631970	230968
668067	783902
-39958	30379
-789914	294812
713604	983901
822512	246266
963911	18741
-588123	613443
-983466	786942
920745	936464
532021	568434
-337405	752853
104146	792811
549927	99078
798251	812682
483532	635194
-227412	599106
750497	703581
717266	203399
-962524	640496
733515	172517
251824	26274
554035	130420
-919327	196699
579431	994951
-739788	478483
-146409	705895
88123	972744
159631	690011
18081	652535
-812993	902402
176339	154227
-292538	224614
-717899	143941
-242337	239725
-95808	979513
-584473	642274
-704947	246750
-529001	406381
-556453	424462
-740753	753808
-97569	930147
-997010	222685
26871	456936
253431	215625
474940	311434
174932	412259
613521	117207
-508502	646208
-609845	719014
-363723	459767
536354	73688
863211	70698
-447385	97570
175499	351001
-143338	342293
607040	33577
713016	647099
65741	155601
341199	765447
-557470	129170
-527548	181876
427772	561439
-583385	8824
195934	184323
24107	327661
-680065	934702
-748281	164070
-933704	229812
281686	87363
865033	644833
-908146	172381
541298	116505
761499	216243
-79270	412177
-264392	436284
308159	116349
-869151	864630
77240	314686
-957282	112724
-119122	977757
-410131	402255
-883015	943553
433689	221405
993186	300675
253358	81419
-841190	389579
-880075	775082
599381	368674
32147	842309
-68626	961431
266290	371563
152685	254578
-155585	204620
-374727	197806
-522397	967516
-106350	808706
-806651	205133
371582	804515
767550	353014
792593	937992
279041	204282
684096	873320
549055	28905
-635418	919985
22455	442382
523614	548733
-414352	355384
80170	243318
114662	527220
-507814	319814
862397	115207
588541	799303
-85615	864710
-787308	500128
-203159	38935
-586525	562550
-294948	976902
208504	573424
442926	204439
-30361	712253
555788	91002
-318200	679544
-437579	765159
-102214	552468
686834	271979
-230187	858504
322272	669804
-286092	878309
493230	321235
743940	351597
-830814	423737
516543	741937
-34279	695868
416724	798082
-815648	1268
-769859	231455
802971	70079
-812984	356172
-912773	365754
216999	109694
749830	940509
890015	973404
193390	7683
-261924	940759
915629	272760
324960	42619
109146	361943
-563136	174927
411785	604052
-98021	337404
-135853	603586
-113773	493601
349981	686992
-765704	465268
-12213	380898
259989	222210
-771923	847709
-837584	410845
-775768	338982
-79706	953356
9689	605561
-880180	719335
240426	69316
-608355	351373
949747	879938
-563897	139927
-737018	428202
-272081	265787
-982071	557907
328816	637613
-12964	163654
817049	43834
631087	284261
-132116	408968
-131117	875067
511014	438965
629473	175983
595640	448065
489003	946488
721348	275304
-423399	804620
731011	138021
274309	769109
886923	417577
-367260	548694
243887	576061
260439	205534
705193	317527
932085	806530
-716884	44230
655647	983982
-758063	231345
420665	505655
367527	908930
239709	276190
503309	36429
689521	296869
-84419	518414
-411030	450500
-716831	683736
297936	339383
-683659	613798
240107	34464
49311	401991
-308796	158052
116974	661361
-17564	867234
-640039	951654
63689	879036
-326572	595867
633684	893803
441130	93815
963568	333922
-508012	899586
-961385	724734
-207299	841708
-489642	375624
-585205	15663
199182	595704
-263657	922276
-528220	72313
132164	29795
866258	509715
-634815	17727
321715	495464
-683018	219115
705681	225110
-595578	326667
953480	525849
-190417	305859
-539573	834079
-298883	482595
-152007	865206
-855404	16373
-974739	854440
302454	537458
988229	243140
-565978	838718
643038	308551
832154	15320
459897	554893
159952	370128
-889801	38487
129364	410244
903005	384983
688461	687437
-209383	192018
955415	274349
544465	433739
659363	265893
-596472	725790
367658	885742
-137791	291895
773129	421259
-941192	840616
-963694	529077
-523653	738461
-843104	693876
-1272	754693
-984342	414056
982731	10528
-52365	894539
-492127	32330
-970375	321812
375218	263004
-165043	743050
158656	266703
860639	109807
-353322	627431
-763392	128125
955706	110857
873809	163222
710358	655349
-838802	142076
388504	33646
-778821	715041
214567	390050
-388773	250689
-289558	120364
428683	883756
981864	839462
61924	229623
552874	456334
559801	295136
921244	199992
-545689	495165
913184	709733
-63733	614858
-840440	904417
244578	333100
-293193	831316
969015	409592
-808059	962466
818804	38620
-487976	959864
190793	505553
125003	935089
932910	998822
507858	839263
719520	600193
-814938	409738
-266168	895106
947021	703165
600559	38321
-65584	526297
214248	233443
374429	358446
-244341	807709
245463	831919
667353	551439
-422343	882729
525000	148898
-37248	612271
28314	729183
57999	794767
-47547	525368
-575352	899797
521893	660490
499959	422306
317288	606011
-251161	28354
-594771	69706
349839	623307
-808071	651621
-304623	225973
-360176	789872
649171	365224
812964	403469
784464	903429
-1	-1
