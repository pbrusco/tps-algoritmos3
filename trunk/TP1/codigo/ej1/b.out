206204
316801
47392
184128
354337
22654
149254
244971
44164
207899
63969
13071
11276
303293
47151
367054
27665
557614
110719
23777
527136
94889
118342
233345
28794
39655
203175
417632
323805
402465
24289
624663
550653
107785
175487
380444
172639
205921
107897
9250
119033
432097
142504
61804
85396
43930
659826
194349
194229
417505
174976
214305
405122
55889
588729
191081
493040
285381
96946
9376
947781
368101
52904
79441
134016
321650
113923
573248
7816
130066
368010
61256
731367
189725
119728
124389
33745
577648
931273
570617
290654
1477
118284
59966
792524
3248
200168
340751
121345
360743
93520
215992
121893
456503
625810
24361
624433
250696
208300
36600
256215
133777
13894
18909
329317
456157
22492
202077
64102
291150
182928
257377
17990
44076
290802
513570
290222
532336
399933
63368
653650
300921
3221
436608
434281
74853
442515
125241
60497
25838
118700
404940
399024
199233
145377
594592
223345
316124
258280
57267
570411
54287
345744
305554
594500
500859
488349
6329
460467
332368
150791
579621
50993
103407
148994
414064
438517
76085
113764
454025
860933
194785
45171
70779
47520
175617
375928
144602
283886
27968
97690
13360
400474
119391
219632
101888
440738
20879
149293
507148
6081
20150
171868
49317
50300
21767
455176
79486
406953
36378
193829
92038
240693
240480
152321
169068
429591
77158
471161
147280
133314
33245
368641
779
151058
374752
814871
497397
361407
756709
115910
17064
4592
176041
52233
202139
2237
245944
23453
273601
127843
251269
99775
140315
66338
694540
348300
471208
696356
533505
213012
226390
366675
19970
310425
447854
261388
303600
15530
644419
328355
28792
305758
228837
185401
96035
336075
325416
314657
57006
254465
67297
40452
21364
124502
138319
728531
70217
15455
9615
84148
22096
39166
438356
435464
41
325242
58627
116393
9001
159416
254460
138609
264546
46179
224719
153401
192446
369282
893112
206016
576033
66752
161175
199317
203520
352931
49678
84936
94081
58033
58672
58543
617936
289766
622914
228158
348754
162979
27824
186516
705251
434938
107327
720873
37837
216744
239086
76365
79715
402295
93361
375865
54853
1262
126925
630969
27468
62766
27784
99292
167872
527373
25077
118447
432945
227278
30889
594276
68
821655
419075
111329
218721
142416
375131
333499
832
445884
324272
130377
152928
120371
143688
209001
168946
141425
287739
129118
199914
376564
171200
25449
6433
163727
661104
409081
151170
113633
8855
209370
463451
240510
11582
343529
527867
133802
6468
279671
561140
574656
222572
179011
921585
27206
518261
205955
172216
66756
287812
561610
450683
62745
730788
126049
446400
10367
125803
92147
969424
450764
150720
223682
439219
84873
723770
187448
855516
54396
108832
188433
238
53338
136724
53866
415517
88283
716608
560415
27684
808960
22412
252146
24251
2690
4023
382701
590485
637362
138753
7960
34112
34859
96513
623713
84809
781158
99810
646372
12441
279966
328417
416872
427192
155018
234739
614925
175088
78444
74637
14034
676
682137
254426
66135
510386
402208
339777
431354
217305
59070
92284
53864
277488
12361
45226
8992
145307
316979
19944
77965
115768
148292
279103
1258
265936
722346
193453
257824
563860
263898
27680
119706
24031
182021
193037
323198
595215
437458
96448
1
297171
112416
92241
108112
21032
2736
232105
232952
251990
321073
70324
204423
215601
18810
708317
94258
111672
86017
470232
400532
122368
233587
24073
277152
847899
198619
16965
167238
428076
179740
211755
227574
28841
332545
10717
32833
433161
191396
46897
39232
16643
118862
280465
156991
83290
122954
539919
483164
37633
2375
903225
378075
829580
70461
287781
112976
70443
438972
82168
80291
293086
387743
84026
538375
492663
95839
66006
670807
81
184931
263541
1
236874
564596
344993
726652
175680
27247
195101
89110
241493
16798
332544
123952
132948
700352
323264
314046
42801
253449
7056
91197
142332
234108
90366
51760
195706
307799
311745
155543
15414
610435
335045
52025
678893
60670
26763
208439
157892
95095
914575
92872
411489
37425
53929
681504
501225
149129
109655
5902
348319
322401
165267
10751
417851
35080
16744
244181
569308
10566
75352
86968
157712
116521
988174
200821
532960
28106
274553
186094
63287
151361
203680
30577
148825
411624
130469
178633
250216
48681
201652
405994
134981
406413
512517
600743
277377
130994
341139
277081
207181
247171
77176
389092
385492
300316
80509
204900
278305
92352
601251
377595
501933
110278
443610
112967
233036
6057
10879
71882
823860
315513
37249
252132
31976
648727
481009
72099
2912
131168
231085
18647
196876
251460
208146
6077
33936
731862
53601
360109
349727
132544
35133
483445
32396
325380
374558
54175
70720
43447
4204
2645
64168
774765
217178
634476
130177
624888
400146
492365
9812
93120
53039
213493
957005
413969
52249
283761
65967
674721
533694
142535
406809
106251
94608
350442
87156
522240
488787
250028
18376
22233
83095
120682
73886
2654
65917
111131
400455
100050
22100
427772
8481
102037
24107
73721
163521
77884
58807
529320
126789
20168
205416
258020
421624
3875
43551
60874
112104
439859
370124
286095
62274
194901
59073
327547
358043
254469
370000
168056
118107
238031
84605
172981
151089
402634
13881
280488
293390
1
181117
273056
18190
38147
14563
537021
165720
142906
359596
197022
20299
588541
230605
42944
28946
21975
748116
9344
34048
283747
21382
60256
132778
68488
207811
608577
57600
592217
14760
344811
152968
520029
31405
599242
816
156661
32102
484
188947
28401
88552
652645
6976
678835
123361
23420
109146
131826
201493
112717
188487
318154
148929
231468
373689
81621
112583
311131
333886
359952
575694
54280
18300
337898
199837
94882
210124
259493
26538
339463
15902
28499
62565
252816
461446
56964
58924
349730
338145
125440
70741
24565
483775
51769
208494
44234
204075
157324
728915
43986
42201
104752
224440
793849
123861
10580
158411
146493
19500
287665
142088
251423
29089
76779
60252
116974
85384
895447
871705
538127
290532
90295
80476
846370
672085
801389
177120
4541
529728
406521
50284
24579
302822
1794
448209
126293
187961
300231
48560
80891
831658
44848
860427
3071
31161
383226
127501
349284
25936
11416
459897
190576
9194
79768
116478
1024
64489
132368
16575
186995
705444
389986
143519
225198
11816
354074
381451
377608
711254
188560
2241
75619
12519
165113
8464
628849
164038
91990
451029
101068
81991
98689
410143
80072
7644
92664
111499
118497
84468
98705
447148
12148
339420
222113
123280
52766
438705
225897
478947
33776
155285
212153
764301
36596
498552
258288
399631
970552
542907
573814
310396
214132
360771
25744
460713
93988
288739
229139
6990
281457
801729
61098
51423
411060
660885
98057
140838
314079
303341
458016
10491
23711
32595
458199
80652
337248
213785
328212
666036
