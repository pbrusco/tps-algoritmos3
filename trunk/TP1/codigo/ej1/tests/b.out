65569
149794
7345
461017
923105
231959
27073
252449
675949
8245
394540
88359
55442
119050
469302
5640
252505
677941
83587
236516
142129
627399
719029
1491
790316
32058
638409
54063
274784
117169
472479
34709
485638
87256
446040
65233
385944
631105
74625
23025
6737
54080
79324
226422
116663
74349
38250
341840
541932
291939
359253
729394
77557
131076
248943
26737
288425
583656
229345
224532
614319
7315
541443
83440
228752
110646
12616
5609
27603
351729
300213
30691
270657
254884
258852
27223
378960
152030
78336
293671
20001
159403
689280
384186
60245
143595
132562
138116
366816
93994
167240
16903
424321
109328
25624
457029
13737
385425
226839
293431
60232
294556
354509
60683
76843
327216
1
10957
369619
438156
70638
184050
652164
461924
16384
528
239665
19463
45460
133437
151051
29337
7393
175663
231868
81424
125921
103395
689181
587114
58871
10045
212096
311476
148280
32225
41202
17156
281799
260963
170479
167144
410041
293879
3659
137230
98157
417793
7614
39202
